lua/neo-tree/command/init.lua

```diff
@@ -219,14 +219,13 @@ handle_reveal = function(args, state)
   elseif not utils.is_subpath(cwd, path) then
     -- force was not specified, so we need to ask the user
     cwd, _ = utils.split_path(path)
-    inputs.confirm("File not in cwd. Change cwd to " .. cwd .. "?", function(response)
-      if response == true then
-        args.dir = cwd
-      else
-        args.reveal_file = nil
-      end
-      do_show_or_focus(args, state, true)
-    end)
+    local response = vim.fn.confirm("File not in cwd. Change cwd to " .. (cwd or "nil") .. " ?", "&y[Y]\n&n[N]")
```

Comment:
This is entirely the wrong place to do this. This sort of change should go into the `inputs.confirm` function so that it applies to all confirmation prompts. It should be in the `else` branch of `use_popups_for_input()` check since this will not be a popup by default.